pkg/cmd/controller/csr.go

```diff
@@ -0,0 +1,51 @@
+package controller
+
+import (
+	"context"
+
+	"github.com/openshift/library-go/pkg/operator/csr"
+	"github.com/openshift/library-go/pkg/operator/events"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/selection"
+)
+
+const (
+	controllerName = "csr-approver-controller"
+	// Hardcoded for the Monitoring Operator:
+	// https://github.com/openshift/cluster-monitoring-operator/blob/e7a2b2e20d19ba7a9271df59e06b1fd32b8bdfe4/manifests/0000_50_cluster-monitoring-operator_03-role-binding.yaml#L25-L27
```

Comment:
nittiest of nits: let's not put github references in the code, I think the hardcode is apparent here.

btw: I believe the correct reference would be https://github.com/openshift/cluster-monitoring-operator/blob/e7a2b2e20d19ba7a9271df59e06b1fd32b8bdfe4/manifests/0000_50_cluster-monitoring-operator_03-role-binding.yaml#L1-L10

Reply:
Removed that comment.